packages/ua-devtools-solana/src/oft/sdk.ts

```diff
@@ -574,11 +574,8 @@ export class OFT extends OmniSDK implements IOApp {
         return web3Transaction
     }
 
-    protected async _getOwnerSigner(): Promise<Signer> {
+    protected async _getAdmin(): Promise<Signer> {
         const owner = await this.getOwner()
-        if (!owner) {
-            throw new Error(`No owner found for ${this.label}`)
-        }
         return createNoopSigner(publicKey(owner))
```

Comment:
Would `createNoopSigner(publicKey(owner))` throw an error if `owner` was falsey since we are removing the validation check?

Reply:
Good question.  It must exist in the contract.  If falsey is returned, we have bad info.